internal/component/otelcol/receiver/awsecscontainermetrics/awsecscontainermetrics.go

```diff
@@ -0,0 +1,86 @@
+// Package awsecscontainermetrics provides an otelcol.receiver.awsecscontainermetrics component.
+package awsecscontainermetrics
+
+import (
+	"time"
+
+	"github.com/grafana/alloy/internal/component"
+	"github.com/grafana/alloy/internal/component/otelcol"
+	otelcolCfg "github.com/grafana/alloy/internal/component/otelcol/config"
+	"github.com/grafana/alloy/internal/component/otelcol/receiver"
+	"github.com/grafana/alloy/internal/featuregate"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecscontainermetricsreceiver"
+	otelcomponent "go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pipeline"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:      "otelcol.receiver.awsecscontainermetrics",
+		Stability: featuregate.StabilityExperimental,
+		Args:      Arguments{},
+
+		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			fact := awsecscontainermetricsreceiver.NewFactory()
+			return receiver.New(opts, fact, args.(Arguments))
+		},
+	})
+}
+
+// Arguments configures the otelcol.receiver.awsecscontainermetricsreceiver component.
+type Arguments struct {
+	CollectionInterval time.Duration `alloy:"collection_interval,attr,optional"`
+
+	DebugMetrics otelcolCfg.DebugMetricsArguments `alloy:"debug_metrics,block,optional"`
+
+	// Output configures where to send received data. Required.
+	Output *otelcol.ConsumerArguments `alloy:"output,block"`
+}
+
+var _ receiver.Arguments = Arguments{}
+
+// SetToDefault implements syntax.Defaulter.
+func (args *Arguments) SetToDefault() {
+	*args = Arguments{
+		// TODO: Let default be set by otel component?
+		// CollectionInterval: 20 * time.Second,
+	}
```

Comment:
```suggestion
    	cfg := awsecscontainermetricsreceiver.NewFactory().CreateDefaultConfig().(*awsecscontainermetricsreceiver.Config)
   	*args = Arguments{
		CollectionInterval: cfg.CollectionInterval,
	}
```

We could be explicit here and take the default from otel.

Reply:
done - I've set the default to explicitly reference the underlying OTEL component default.